version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.14.4"
+version in ThisBuild := "0.14.5"
```

Comment:
the change of non-internal interface `McSsoJwtRequestRest` would qualify this for a minor bump. Let's change it to `0.15.0`

Reply:
Set to 0.15.0 as requested.